Packs/GenericSQL/Integrations/GenericSQL/GenericSQL.py

```diff
@@ -279,24 +280,21 @@ def sql_query_execute(client: Client, args: dict, *_) -> Tuple[str, Dict[str, An
         skip = int(args.get('skip', 0))
         bind_variables_names = args.get('bind_variables_names', "")
         bind_variables_values = args.get('bind_variables_values', "")
-        bind_variables = generate_bind_vars(bind_variables_names, bind_variables_values)
+        bind_variables = generate_bind_vars(bind_variables_names, bind_variables_values, sql_query)
 
         result, headers = client.sql_query_execute_request(sql_query, bind_variables)
-        # converting an sqlalchemy object to a table
-        converted_table = [dict(row) for row in result]
-        # converting b'' and datetime objects to readable ones
-        table = [{str(key): str(value) for key, value in dictionary.items()} for dictionary in converted_table]
-        table = table[skip:skip + limit]
-        human_readable = tableToMarkdown(name="Query result:", t=table, headers=headers,
-                                         removeNull=True)
+        result = result[skip:skip + limit]
```

Comment:
why change `table` to `result`? I mean, it's better indeed, but was there a reason?

Reply:
I renamed this variable after changing the code around this line.
I think it's better